lib/email_address.rb

```diff
@@ -13,59 +13,28 @@ module EmailAddress
     require "email_address/canonical_email_address_type"
   end
 
+  class << self
+    %i(valid? error normal redact munge canonical reference).each do |proxy_method|
+      define_method(proxy_method) do |*args, &block|
+        EmailAddress::Address.new(*args).send(proxy_method, &block)
```

Comment:
[`public_send`](http://ruby-doc.org/core-2.4.2/Object.html#method-i-public_send) will be better and more accurate.